### DISCUSSION TITLE
creating buttons to mark gas & air changes for Diedrich

### DISCUSSION THREAD

**Discussion by elguapo1911** [artisan-roaster-scope/artisan#849] [Q&A] [answered]
New to artisan, just watched the webinar from last Friday about the update, wow! Great info. I need some guidance on adding buttons (?) so that I can manually mark gas % changes & air changes on my Diedrich IR12 non-automated machine. I'd like to set them up similar to the ones in the webinar with the color changing as percentage of power changes. Thank you in advance!

**Comment by roasterdave** [comment, marked as answer]
Buttons can be configured on the **Config>> Events>> Buttons tab**.  After you add a button you can click on the box in the **Color** column to change the button's color.   A good resource to learn about configuring [buttons and events](https://artisan-scope.org/docs/events/) is the [Quick Start Guide](https://artisan-scope.org/docs/quick-start-guide/) found at [artisan-scope.org](https://artisan-scope.org).

Feel free to come back with any specific questions.

**Reply by elguapo1911** [reply]
Thank you! I wonder if there's a way to back into recording energy used depending on gas setting? If I knew the amount of btu's at 100%...would be helpful to figure our how much propane I'm using. Thanks again

**Reply by roasterdave** [reply]
The Energy Calculator will estimate your energy consumption and CO2 production based on the gas settings recorded in a roast.  A complete description of the Energy Calculator with examples can be found on the [Artisan Blog](https://artisan-roasterscope.blogspot.com/2021/07/tracking-energy-consumption-co2.html).

**Comment by fcebreros** [comment]
Check under
Roasts/Properties under the Energy tab.
